src/services/network.service.ts

```diff
@@ -277,12 +282,37 @@ export class NetworkService implements OnApplicationBootstrap {
     };
   }
 
+  closeExpiredStateChannelsAction() {
+    const config = NETWORK_CONFIGS.kepler;
+    const client = new GraphqlQueryClient(config);
+
+    return async () => {
+      const apolloClient = client.explorerClient;
+      const now = new Date();
+      const indexer = await this.accountService.getIndexer();
+      const result = await apolloClient.query<
+        GetIndexerUnfinalisedPlansQuery,
+        GetIndexerUnfinalisedPlansQueryVariables
+      >({
+        query: GetIndexerUnfinalisedPlans,
+        variables: { indexer, now },
+      });
+
+      const unfinalisedPlans = result.data.stateChannels.nodes;
+
+      unfinalisedPlans.forEach(async (node) => {
+        await this.sdk.stateChannel.claim(node.id);
```

Comment:
Use `sendTransaction` function to send the each tx, which can inject logs and collect the failed txs to a pool then resend these txs

```
await this.sendTransaction(`claim plan for ${node.consumer} with status....`, async () =>
await this.sdk.stateChannel.claim(node.id);,
);
```